packages/workflow/src/Interfaces.ts

```diff
@@ -2284,7 +2284,7 @@ export interface IWorkflowExecutionDataProcess {
 	executionData?: IRunExecutionData;
 	runData?: IRunData;
 	pinData?: IPinData;
-	retryOf?: string;
+	retryOf?: string | null;
```

Comment:
I thought it was `string | undefined`? If `null` is what the DB returns, can we transform it to `undefined` when we retrieve?

Having to remember `retryOf ?? undefined` disregarding the current type is an accident waiting to happen.

Reply:
I want to spend more time on `retryOf` in a separate PR, to make the types around this more reliable.